bootstrap/utils/logging.go

```diff
@@ -0,0 +1,60 @@
+package utils
```

Comment:
Ahhhg! This causes a cyclic import. :-(

Please create a logging package to avoid this and name the file logging/logrus-adaptor.go

Reply:
yeah... i didn't spot it at first. I'll work through it.